packages/functional-tests/src/app.ts

```diff
@@ -32,16 +34,18 @@ export default class App {
     /**
      * Registry of functional tests. Add your test here.
      */
-    private testFactories: { [key: string]: (user: MRESDK.User) => Test } = {
+    private testFactories: { [key: string]: () => Test } = {
         'gltf-animation-test': (): Test => new GltfAnimationTest(this, this.baseUrl),
-        'look-at-test': (user: MRESDK.User): Test => new LookAtTest(this, this.baseUrl, user),
+        'look-at-test': (): Test => new LookAtTest(this, this.baseUrl),
         'rigid-body-test': (): Test => new RigidBodyTest(this),
         'text-test': (): Test => new TextTest(this),
         'clock-sync-test': (): Test => new ClockSyncTest(this, this.baseUrl),
+        'library-test': (): Test => new LibraryTest(this, this.baseUrl),
         'primitives-test': (): Test => new PrimitivesTest(this, this.baseUrl),
         'input-test': (): Test => new InputTest(this, this.baseUrl),
+        'gltf-gen-test': (): Test => new GltfGenTest(this, this.baseUrl),
         'root-motion-test': (): Test => new RootMotionTest(this, this.baseUrl),
-        'asset-preload-test': (user: MRESDK.User): Test => new AssetPreloadTest(this, this.baseUrl, user)
+        'asset-preload': (): Test => new AssetPreloadTest(this, this.baseUrl)

```

Comment:
Looks like a regression here. should be `asset-preload-test`

Reply:
no, this was actually not a regression ,this was because the fixes was in the revert-revert-revert. Don't worry about this one

Reply:
Good catch!